src/components/cylc/analysis/TimeSeries.vue

```diff
@@ -126,6 +126,15 @@ query analysisJobQuery ($workflows: [ID], $tasks: [ID]) {
 }
 `
 
+/** The one-off query which retrieves historical tasks names */
+const TASK_NAMES_QUERY = gql`
+query analysisTaskNamesQuery ($workflows: [ID]) {
```

Comment:
There is no match for this in the mock data which is why the E2E tests are failing. This should sort it
```suggestion
query analysisTaskQuery ($workflows: [ID]) {
```